README.rst

```diff
@@ -19,6 +19,10 @@ python-can
    :target: https://ci.appveyor.com/project/hardbyte/python-can/history
    :alt: AppVeyor CI Server for develop branch
 
+.. |coverage| image:: https://codecov.io/gh/hardbyte/python-can/branch/master/graph/badge.svg
```

Comment:
I think this should point to `develop` coverage (as with travis).

Reply:
Yes, done.